### PR TITLE
review changes for sayt

### PR DESCRIPTION
**Pull request by generall**
TBD:

- Use new design in images
- Preview image with new style as well
- Icon

### CODE REVIEW

File: qdrant-landing/content/articles/sayt.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ keywords: search, semantic, vector, llm, integration, benchmark, recommend, perf
 
 Qdrant is one of the fastest vector search engines out there, so while looking for a demo to show off, we came upon the idea to do a search-as-you-type box with a fully semantic search backend. Now we already have a semantic/keyword hybrid search on our website. But that one is written in Python, which incurs some overhead for the interpreter. Naturally, I wanted to see how fast I could go using Rust.
 
-Since Qdrant doesn't embed by itself, I had to decide on an embedding model. The prior version used the [SentenceTransformers](https://www.sbert.net/) package, which in turn employs Hugging Face's Bert-based [All-MiniLM-L6-V2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/tree/main) model. This model is battle-tested and delivers fair results at speed, so not experimenting on this front I took an [ONNX version](https://huggingface.co/optimum/all-MiniLM-L6-v2/tree/main) and ran that within the service.
```
Comment (generall) [inline qdrant-landing/content/articles/sayt.md:17]: Huggingface only hosts it, they didn't train it

File: qdrant-landing/content/articles/sayt.md [hunk 2]
```diff
@@ -23,45 +23,14 @@ The workflow looks like this:
...
-      "score": 1.0,
-      "vector": null,
-      "version": 1
-    },
-    ... (4 more entries omitted)
-  ],
-  "status": "ok",
-  "time": 0.002377142
-}
-```
```
Comment (generall) [inline qdrant-landing/content/articles/sayt.md:63]: Too long, not important for the story

File: qdrant-landing/content/articles/sayt.md [hunk 3]
```diff
@@ -71,68 +40,32 @@ Making that work requires setting up the `prefix_cache` collection with points t
...
-      "score": 0.7441973,
-      "vector": null,
-      "version": 0
-    },
-    ... (4 more entries omitted)
-  ],
-  "status": "ok",
-  "time": 0.000872409
-}
-```
```
Comment (generall) [inline qdrant-landing/content/articles/sayt.md:114]: Same, too long

File: qdrant-landing/content/articles/sayt.md [hunk 4]
```diff
@@ -71,68 +40,32 @@ Making that work requires setting up the `prefix_cache` collection with points t
...
-
-Inspired by a [recent article I wrote](https://qdrant.tech/articles/io_uring/), I chose the search terms "io", "ring" and "io_uring", which apart from all finding the article in question, trigger different code paths in both versions.
+Since the operations themselves are far faster than the network whose fickle nature would have swamped most measurable differences, I benchmarked both the Python and Rust services locally. I'm measuring both versions on the same AMD Ryzen 9 5900HX with 16GB RAM running Linux. The table shows the average time and error bound in milliseconds. I only measured up to a thousand concurrent requests. None of the services showed any slowdown with more requests in that range. I do not expect our service to become DDOS'd, so I didn't benchmark with more load.
 
 Without further ado, here are the results:
 
-![Benchmark Results](/articles_data/sayt/benchmark.png)
-
-| query     | `io`        | `ring`    | `io_uring` |
```
Comment (generall) [inline qdrant-landing/content/articles/sayt.md:128]: `io_uring` is still confusing

**Comment by llogiq** [comment]
Thanks for the review! I'll look into it.

File: qdrant-landing/content/articles/sayt.md (hunk 4 above)
Comment (llogiq) [reply qdrant-landing/content/articles/sayt.md:128]: Even with the explanation above?

**Pull request merged**